pandas/tests/frame/indexing/test_where.py

```diff
@@ -879,6 +880,7 @@ def test_where_nullable_invalid_na(frame_or_series, any_numeric_ea_dtype):
         OPTIONAL_DICTS, OPTIONAL_FLOATS, OPTIONAL_INTS, OPTIONAL_LISTS, OPTIONAL_TEXT
     )
 )
+@settings(deadline=None)  # GH 44969
```

Comment:
can you move the hypothesis setup to the  pandas/_testing/_hypothesis.py instead (and make this change)